Helper/Webhook.php

```diff
@@ -340,11 +172,8 @@ public function processNotification(Notification $notification): bool
 
             $transitionState = $this->getTransitionState($notification, $currentState);
 
-            if ($transitionState !== $currentState) {
-                $this->handleOrderTransition($notification, $transitionState);
-            } else {
-                $this->handleUnchangedStates($this->order, $notification);
-            }
+            $webhookHandler = self::$webhookHandlerFactory::create($notification->getEventCode());
```

Comment:
This will throw the new exception type. Consider removing the small try/catch below

Reply:
https://github.com/Adyen/adyen-magento2/pull/1628/commits/79447e38f543a3b660f942dca8b476030e0ec955